src/views/expired-password/Footer.js

```diff
@@ -63,6 +65,9 @@ export default View.extend({
     },
   },
   getTemplateData: function() {
-    return { passwordWarn: this.options.appState.get('isPwdExpiringSoon') };
+    return {
+      passwordWarn: this.options.appState.get('isPwdExpiringSoon'),
+      showSignOutLink: !this.settings.get('features.hideBackToSignInForReset')
```

Comment:
shall add feature to  `models/settings.js`

Reply:
> shall add feature to `models/settings.js`

Hi @haishengwu-okta, previously @oleksandrpravosudko-okta has told me to use this flag which is an existing one already exist in `model/settings.js` It has been there since [this commit](https://github.com/okta/okta-signin-widget/commit/06e217bd33eaddb1ef3a9f3c172c5ee6fa57b39b#diff-1ae33f896687ee76b3e074ce7fd60b12b15c9521b183805382771878e0ff7355R82) made in 2019.